components/main-chef-wrapper/main.go

```diff
@@ -42,6 +47,34 @@ func createDotChef() {
 	os.Mkdir(path, 0700)
 }
 
+func createRubyEnv(){
+//	get installation path
+//	homepath, err := homedir.Dir()
+//	if err != nil {
+//		log.Fatalf(err.Error())
+//	}
+//	todo ==> incase home directory is needed we can add it to string
+	installationPath :=  platform_lib.ExpectedOmnibusRoot() + "/ruby-env.json"
+	fmt.Printf(installationPath)
+	result, err := exists(installationPath)
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+	if result == true {
+		fmt.Print("file exists======== ruby script not needed")
```

Comment:
I think we have to take this a step further.  The contents of this file can change across version, so we'll need to regenerate it when a new version of WS is in use. The easiest way I can think of to address that is to capture the workstation version in the JSON file, and compare it if the file is present.

Reply:
addressing in commit https://github.com/chef/chef-workstation/pull/2430/commits/5ed7f94e99ce333eb65021b553d5b327cc33aae8